### DISCUSSION TITLE
About Submodding

### DISCUSSION THREAD

**Discussion by MoriMoriMorii** [Monika-After-Story/MonikaModDev#8625] [general - submod / external content]
I'm making a submod storyline about chibika because chibika best chibi and because I want to show my take on what I think can happen (So an alternate timeline) I really wanted to use monika's normal DDLC sprite and I didn't know if that was allowed.
(TLDR: Can I use OG Monika sprite from base DDLC)

**Comment by multimokia** [comment]
You _can_ however they're much more limited than the MAS sprites. You'll want to set the `is_sitting` flag to `False`, this will make Monika use her stock standing sprite.

The corresponding spritecodes are a little awkward to get working since not _every_ spritecode MAS has, will have an equivalent in standing, you might want to experiment to get em working

**Reply by MoriMoriMorii** [reply]
Alright, Thanks!

**Reply by MoriMoriMorii** [reply]
I'm having a few difficulties getting Monika's standing sprites to show.
I did have an idea however,
Would I be allowed to add the base Monika sprites into the mod_aseets and re-define them to use different variables? or is there something I can do to make them display

**Comment by MoriMoriMorii** [comment]
Would it be alright if I re-defined monika images (Of the base game) as monika1
such as
`image monika1 1 = im.Composite((960, 960), (0, 0), "mod_assets/monika1/1l.png", (0, 0), "mod_assets/monika1/1r.png", (0, 0), "mod_assets/monika1/a.png")`
